astropy/utils/tests/test_decorators.py

```diff
@@ -554,3 +554,40 @@ class TestClass(object):
         pass
 
     assert inspect.getdoc(TestClass) == 'what we do is strange.'
+
+
+def test_replaced_parameter():
+    @replaced_parameter('clobber', 'overwrite', since='1.3')
+    def test(a, overwrite=False, c=10):
+        """A simple test function
```

Comment:
Oh, I totally forgot that the `versionchanged` is only appended if there is already a docstring. Sorry for the noise.